sycl/unittests/pi/piPlatformTest.cpp

```diff
@@ -0,0 +1,84 @@
+//==---- piPlatformTest.hpp --- PI unit tests ------------------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "CL/sycl/detail/pi.hpp"
+#include <CL/sycl.hpp>
+#include <gtest/gtest.h>
+#include <memory>
+
+using namespace cl::sycl;
+
+namespace {
+class piPlatformTest : public ::testing::Test {
```

Comment:
It is a common naming pattern in googletest to have `Test` at the end of the test fixture name.
Would it be better to use "PascalCase", i.e., `PiPlatformTest`for the class?

A possible argument for appending the `Test` to the name is that it makes it easy to search for all tests affecting `piPlatform` by searching for that string. It might require excluding the `unittest` path from searches in the non-test codebase though.

Reply:
how about avoiding "pi" prefix at all in class/fucntion names, but define them in "pi" namespace, e.g. pi::platformTest.

Reply:
Good idea! pi::PlatformTest it is. I have also renamed the source file to suit, so it is now pi/PlatformTest.cpp